packages/hashtower/src/types/index.ts

```diff
@@ -0,0 +1,11 @@
+export type HashFunction = (BigInts: BigInt[]) => BigInt
```

Comment:
Can these types be inlined/exported in the main class file?

Reply:
Inlined.